.travis.yml

```diff
@@ -60,6 +60,7 @@ script:
     - mkdir build && cd build
     - ../configure $BITCOIN_CONFIG_ALL $BITCOIN_CONFIG || ( cat config.log && false)
     - make $MAKEJOBS $GOAL || ( echo "Build failure. Verbose build follows." && make $GOAL V=1 ; false )
```

Comment:
@theuni I'd rather have it build the distributed files, to catch errors like  #9393